ISISComputingGroup/ExperimentDatabasePopulator
# Ticket 4402: Fix RB Number populator

See https://github.com/ISISComputingGroup/IBEX/issues/4402

This adds a script that makes it easily runnable in control-svcs and allows it to run in a mode where it only runs once.

To test:
* Set debug to true
* Run main and confirm it pushes data and then stops
* Run main with the --cont flag and confirm it runs continously
